src/main/proto/README.md

```diff
@@ -0,0 +1,18 @@
+# Google Protocol Buffers
+
+Mesos-utils is using Google [Protocol Buffers](https://developers.google.com/protocol-buffers) to marshal and unmarshal data from the persistent store.
+
+## Dependencies
+
+- Google [Protocol Buffers](https://developers.google.com/protocol-buffers) v2.5 should be installed on the system.
+-- Note: protoc 2.6 (and higher) will generate incompatible code. Make sure your version of protoc is 2.5.
+
+## Rebuilt the protos
```

Comment:
Probably. Copied that from https://github.com/mesosphere/marathon/blob/master/src/main/proto/README.md